configs/server.toml

```diff
@@ -180,6 +180,28 @@ certificate = "certs/iggy.pfx"
 # Password for the TLS certificate, required for accessing the private key.
 password = "iggy123"
 
+# Configuration for the TCP socket
+[tcp.socket]
+# Whether to use an ipv4 or ipv6 socket
+ipv6 = false
+
+# Optional, SO_RCVBUF: maximum size in bytes of the receive buffer, can be clamped by the OS
+recv_buffer_size = 102400
```

Comment:
With the `IggyByteSize` (as mentioned below), you can write e.g. `recv_buffer_size = 100 KB`, so it's the same convention as in other places e.g. `QuicConfig`.